client/my-sites/domains/domain-search/index.jsx

```diff
@@ -158,7 +162,23 @@ class DomainSearch extends Component {
 				<span>
 					<div className="domain-search__content">
 						<PlansNavigation cart={ this.props.cart } path={ this.props.context.path } />
-
+						{ hasCustomDomain( selectedSite ) && (
+							<Notice
+								status="is-warning"
+								showDismiss={ false }
+								text={
+									translate( "You're purchasing an additional domain for %(title)s.", {
+										args: { title: selectedSiteTitle },
+									} ) +
+									' ' +
+									translate( 'Would you like to create a new site instead?' )
+								}
+							>
+								<NoticeAction href="/jetpack/new?ref=calypso-selector">
+									{ translate( 'Create new site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Create New Site' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/34990)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).